bundled/tool/_lsp_server.py

```diff
@@ -7,16 +7,32 @@
 class DummyDataCatalog(DataCatalog):
```

Comment:
It may worth to check if it still need to inherit from `DataCatalog`

Reply:
Good point we don't anymore.